.circleci/workflows.yml

```diff
@@ -234,6 +234,17 @@ jobs:
       - run: bash $OPTIMISM_FOLDER/optimism.sh e2e-test
       - run: bash $OPTIMISM_FOLDER/optimism.sh down
 
+  e2e-test-optimism-fallback:
+    executor: vm
+    environment:
+      OPTIMISM_FOLDER: "docker/optimism"
+    steps:
+      - initialize-environment
+      - run: bash $OPTIMISM_FOLDER/optimism.sh up
+      - run: bash $OPTIMISM_FOLDER/optimism.sh deploy-beamer
+      - run: bash $OPTIMISM_FOLDER/optimism.sh e2e-test-fallback
+      - run: bash $OPTIMISM_FOLDER/optimism.sh down
+
```

Comment:
This is wasteful. Why not just add `e2e-test-fallback` to the existing `e2e-test-optimism`?

Reply:
added